crates/vm/wasm/src/gatekeeper.rs

```diff
@@ -763,59 +763,60 @@ mod tests {
 
     #[test_case(
         br#"
-            (module
-                (func (export "sum") (param i32 i32) (result i32)
-                    local.get 0
-                    local.get 1
-                    i32.add
-                ))
-            "#,
+          (module
+            (func (export "sum") (param i32 i32) (result i32)
+              local.get 0
+              local.get 1
+              i32.add
+            ))
+        "#,
         |result| {
             assert!(result.is_ok());
         };
-        "valid_wasm_instance_sanity"
+        "valid wasm instance sanity"
```

Comment:
Test name should be a human readable string (see the example usage in test-case: https://docs.rs/test-case/latest/test_case/#example-usage), so no need for the underscores